src/us/kbase/workspace/kbase/InitWorkspaceServer.java

```diff
@@ -488,56 +484,31 @@ private static AuthToken getHandleToken(
 		return null;
 	}
 	
-
-	private static void checkHandleServiceConnection(
+	private static AbstractHandleClient getHandleServiceClient(
 			final URL handleServiceUrl,
 			final AuthToken handleMgrToken,
 			final InitReporter rep) {
+		final AbstractHandleClient cli;
 		try {
-			final AbstractHandleClient cli = new AbstractHandleClient(
+			cli = new AbstractHandleClient(
 					handleServiceUrl, handleMgrToken);
 			if (handleServiceUrl.getProtocol().equals("http")) {
 				rep.reportInfo("Warning - the Handle Service url uses insecure http. " +
 						"https is recommended.");
 				cli.setIsInsecureHttpConnectionAllowed(true);
 			}
-			cli.isOwner(new LinkedList<String>());
 		} catch (Exception e) {
-			if (!(e instanceof ServerException) || !e.getMessage().contains(
-							"can not execute select * from Handle")) {
-				rep.reportFail("Could not establish a connection to the Handle Service at "
-						+ handleServiceUrl + ": " + e.getMessage());
-			}
-		}
-	}
-	
-	private static HandleMngrClient getHandleManagerClient(
-			final URL handleManagerUrl,
-			final AuthToken handleMgrToken,
-			final InitReporter rep) {
-		final HandleMngrClient cli;
-		try {
-			cli = new HandleMngrClient(
-					handleManagerUrl, handleMgrToken);
-			if (handleManagerUrl.getProtocol().equals("http")) {
-				rep.reportInfo("Warning - the Handle Manager url uses insecure http. " +
-						"https is recommended.");
-				cli.setIsInsecureHttpConnectionAllowed(true);
-			}
-		} catch (Exception e) {
-			rep.reportFail("Could not establish a connection to the Handle Manager Service at "
-					+ handleManagerUrl + ": " + e.getMessage());
+			rep.reportFail("Could not establish a connection to the Handle Service at "
```

Comment:
This message is misleading - initializing the client doesn't validate anything. I think an IOException is specified for the constructor but if you look at the client code is never actually thrown. That's why I left the isOwner() call in the same block.

Reply:
👍